package.json

```diff
@@ -9,7 +9,7 @@
     "test": "jest .spec.js --coverage",
     "e2e": "bash e2e/e2e.spec.sh",
     "eslint": "eslint --fix lib/logic/**",
-    "pkg": "pkg . -t node12-alpine-x64,node12-macos-x64,node12-linux-x64,node12-win-x64 --out-path ./dist",
+    "pkg": "pkg . -t node18-alpine-x64,node18-macos-x64,node18-linux-x64,node18-win-x64 --out-path ./dist",
```

Comment:
@andrii-codefresh @denis-codefresh guys I see that the node version in the package.json is limited to `"node": ">=10.15.3"` - any reason not to increase it to node18 like you did in the rest of the places?